docs/service-agent-setup.md

```diff
@@ -2,20 +2,32 @@
 
 For the deployment of the Agentforce Service Agent it is necessary that certain values in the metadata are replaced with record ids from the used org. To achieve this during source deployment this project makes use of the [string replacement functionality of the Salesforce CLI](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_string_replace.htm). For this certain values will be exported as environment variables.
 
-> [!NOTE]
-> The current instructions assume users of macOS or Linux that have also `jq` for JSON parsing on the command line installed.
-> The final instructions will be inclusive for other operating systems/no usage of `jq` or similar tools.
```

Comment:
@muenzpraeger it's good that you removed jq from the requirements but shouldn't we remove it from the setup script as well?
It's risky to maintain two different versions of the install process.